python-avd/pyavd/_eos_designs/structured_config/network_services/utils.py

```diff
@@ -686,6 +686,52 @@ def get_internet_exit_nat_pool_and_profile(
     def _filtered_internet_exit_policy_types(self: AvdStructuredConfigNetworkServices) -> list:
         return sorted(set(internet_exit_policy["type"] for internet_exit_policy in self._filtered_internet_exit_policies))
 
+    @cached_property
+    def _l3_interface_acls(self: AvdStructuredConfigNetworkServices):
+        """
+        Returns a dict of
+            <hostname> : {
```

Comment:
Since this code runs per device, we should not index acls for interfaces that do not belong to this device.

Reply:
Fixed.